index.bs

```diff
@@ -9911,7 +9911,37 @@ validity of a given {{AudioWorkletProcessor}} subclass.
 			<em>Return type:</em> {{boolean}}
 		</div>
 </dl>
-	
+
+<pre class="example" highlight="js" title="Subclassing AudioWorkletProcessor">
+class MyProcessor extends AudioWorkletProcessor {
+	static get parameterDescriptors() {
```

Comment:
Not required to change, but it would be nice if we didn't indent by 8.  The rendered output indents by two anyway, so might as well make the source code look more like the final output.

Reply:
Done.